deployment-verification/README.md

```diff
@@ -0,0 +1,93 @@
+# Deployment and testing strategies
+
+### Blue-Green deployment
```

Comment:
My understanding of the blue-green deployment is that we would start with 2 environments both having v1 of the service deployed. Then we would release v2 to the blue environment, test and verify it, and if it seem alright, redirect all service trafic from the green env to the blue one. This is when the colors would "switch", so that the env serving live trafic is always called green, and the one where upgrades and verification should be done is called blue.

Could you maybe update the description to better reflect the above?